setup.py

```diff
@@ -17,15 +17,19 @@ def get_readme_md_contents():
 
 install_reqs = ["decorator>=3.3.2", "requests>=2.6.0"]
 
-if [sys.version_info[0], sys.version_info[1]] < [2, 7]:
+if sys.version_info < (2, 7):
     install_reqs.append("argparse>=1.2")
 
+if sys.version_info < (3, 5):
```

Comment:
oh hmm I see you added this here. doesn't seem to be ACK'd when installing via pip. I'll have to admit I forget how pypi/pip uses setup.py. I think this would work fine if you're calling `python setup.py (build|install)` but probably not via pip install directly

Reply:
Interesting, thanks for pointing out this issue! I created #571 so that package maintainers are aware of the issue.